samples/sensor/stm32_temp_sensor/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  description: Usage of STM32 temperature sensor
+  name: stm32_temp_sensor
+tests:
+  sample.sensor.stm32_temp_sensor:
+    depends_on: adc
+    tags: sensors
+    build_only: true
+    filter: dt_compat_enabled_with_label("st,stm32-adc", "ADC_1")
```

Comment:
For some reason this is not producing expected result.
In order to avoid blocking this to get to DV2.6.0, let's go with:
```
    depends_on: adc
    tags: sensors
    build_only: true
    platform_allow: nucleo_f401re stm32f103_mini
```
We'll try to find a better approach later.

Reply:
Note, you can see effect on twister with :
```
twister -T samples/sensor/stm32_temp_sensor/
```

Reply:
Thanks, done!